apps/condo/migrations/20231023101614-0330_payment_frozeninvoice_payment_invoice_and_more.js

```diff
@@ -0,0 +1,51 @@
+// auto generated by kmigrator
+// KMIGRATOR:0330_payment_frozeninvoice_payment_invoice_and_more:IyBHZW5lcmF0ZWQgYnkgRGphbmdvIDQuMi40IG9uIDIwMjMtMTAtMjMgMDU6MTYKCmZyb20gZGphbmdvLmRiIGltcG9ydCBtaWdyYXRpb25zLCBtb2RlbHMKaW1wb3J0IGRqYW5nby5kYi5tb2RlbHMuZGVsZXRpb24KCgpjbGFzcyBNaWdyYXRpb24obWlncmF0aW9ucy5NaWdyYXRpb24pOgoKICAgIGRlcGVuZGVuY2llcyA9IFsKICAgICAgICAoJ19kamFuZ29fc2NoZW1hJywgJzAzMjlfaW52b2ljZWhpc3RvcnlyZWNvcmRfYW5kX21vcmUnKSwKICAgIF0KCiAgICBvcGVyYXRpb25zID0gWwogICAgICAgIG1pZ3JhdGlvbnMuQWRkRmllbGQoCiAgICAgICAgICAgIG1vZGVsX25hbWU9J3BheW1lbnQnLAogICAgICAgICAgICBuYW1lPSdmcm96ZW5JbnZvaWNlJywKICAgICAgICAgICAgZmllbGQ9bW9kZWxzLkpTT05GaWVsZChibGFuaz1UcnVlLCBudWxsPVRydWUpLAogICAgICAgICksCiAgICAgICAgbWlncmF0aW9ucy5BZGRGaWVsZCgKICAgICAgICAgICAgbW9kZWxfbmFtZT0ncGF5bWVudCcsCiAgICAgICAgICAgIG5hbWU9J2ludm9pY2UnLAogICAgICAgICAgICBmaWVsZD1tb2RlbHMuRm9yZWlnbktleShibGFuaz1UcnVlLCBkYl9jb2x1bW49J2ludm9pY2UnLCBudWxsPVRydWUsIG9uX2RlbGV0ZT1kamFuZ28uZGIubW9kZWxzLmRlbGV0aW9uLlNFVF9OVUxMLCByZWxhdGVkX25hbWU9JysnLCB0bz0nX2RqYW5nb19zY2hlbWEuaW52b2ljZScpLAogICAgICAgICksCiAgICAgICAgbWlncmF0aW9ucy5BZGRGaWVsZCgKICAgICAgICAgICAgbW9kZWxfbmFtZT0ncGF5bWVudGhpc3RvcnlyZWNvcmQnLAogICAgICAgICAgICBuYW1lPSdmcm96ZW5JbnZvaWNlJywKICAgICAgICAgICAgZmllbGQ9bW9kZWxzLkpTT05GaWVsZChibGFuaz1UcnVlLCBudWxsPVRydWUpLAogICAgICAgICksCiAgICAgICAgbWlncmF0aW9ucy5BZGRGaWVsZCgKICAgICAgICAgICAgbW9kZWxfbmFtZT0ncGF5bWVudGhpc3RvcnlyZWNvcmQnLAogICAgICAgICAgICBuYW1lPSdpbnZvaWNlJywKICAgICAgICAgICAgZmllbGQ9bW9kZWxzLlVVSURGaWVsZChibGFuaz1UcnVlLCBudWxsPVRydWUpLAogICAgICAgICksCiAgICBdCg==
+
+exports.up = async (knex) => {
```

Comment:
you need to extend time limit for query, as we have lots of payments

Reply:
Fixed